virtual-contests/pepabo/003/sumitrust2019/D.rb

```diff
@@ -0,0 +1,23 @@
+# https://atcoder.jp/contests/sumitrust2019/tasks/sumitb2019_d
+
+
+
+N = gets.to_i
+S = gets.chomp
+
+ans = 0
+(0..999).each do |x|
```

Comment:
```suggestion
("000".."999").each do |code|
```
こうすればよかった。直下の `code = sprintf("%03d", x)` が不要になる。

ex:
```ruby
> ("00".."19").to_a
=> ["00", "01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19"]
```